Sources/Twift+Lists.swift

```diff
@@ -398,7 +398,7 @@ public struct PinnedResponse: Codable {
 /// A response object pertaining to list created
 public struct CreatedListResponse: Codable {
     /// The ID for the newly-created List
-    public let id: Int
+    public let id: List.ID
     /// If List name was returned, response is considered successfull
```

Comment:
Sam comment applies here as above 😊

```suggestion
    /// The name for the newly-created List
```

Reply:
Right! was wondering and went to check Twitter doc to make sure how optional it is, I remember now paying attention if its optional or not and just copied their description text for it 😅